pyanaconda/modules/payloads/source/harddrive/harddrive.py

```diff
@@ -54,6 +54,11 @@ def __init__(self):
         )
         self._iso_name = ""
 
+    @property
+    def is_iso_mounted(self):
```

Comment:
We can change this in future. But in general I've rather add code than comment to the code. 

For me it's much self explanation to use:
```
if self.is_iso_mounted:
```
than
```
if self._iso_name:
```

The first is check if the iso is mounted so I can react to that.
The second one is check if the iso name is set what can have several reasons so I don't know how to approach this when reading the code. I know it can be explained by comment but then you are using explanation of your code instead of changing the code to be self-explanatory. 

But yes, I agree that both approaches have the same result here. I just like the second one more.